rapidproCSDAdapter.js

```diff
@@ -89,8 +90,9 @@ module.exports = function (config) {
           callback(null, converted, _orchestrations.concat(orchestrations))
         }
       }
-
-      rapidpro.getContacts(groupUUID, getContactsCallback([]))
+      
+		let url = rapidpro.contactsURL(groupUUID)
```

Comment:
Yes,because i will now be looping through the getContacts(),so i just want it to accept a url other than groupUUID